### DISCUSSION TITLE
doris日志一直有backend [10027] invalid situation. tablet[245732] has few replica[1], replica num setting is [1]这类报错

### DISCUSSION THREAD

**Discussion by echobigdata** [apache/doris#25939] [A - General / Q&A]
doris版本2.0.2，看了下是这三个doris内部表table_statistics，column_statistics，histogram_statistics，这种情况如何解决呢，会对正常使用造成影响吗
![image](https://github.com/apache/doris/assets/97231854/ad682a99-97f2-4740-8777-cb3dcb47305b)

**Comment by Jack96125** [comment]
我的也是
